### DISCUSSION TITLE
Tags entrega final - Macowins

### DISCUSSION THREAD

**Discussion by francoparente** [dds-jv/foro#351] [Consultas generales] [answered]
Buenos días,

Quería hacer una consulta con respecto al tag "entrega-final".

Teniendo el tp resuelto (checkeado con ``mvn verify``) y sin committear, por error pusheé el tag de entrega-final con el siguiente comando:

``mvn clean verify && git tag entrega-final && git push origin HEAD --tags``

Después me di cuenta y committeé y pusheé lo que me faltaba. Pero cuando quise tirar el comando anterior me avisó que el tag "**entrega-final**" ya existía, por lo que le cambié el tag a "**entrega-final-REAL**". Me quedaron así los tags en el repo:

![image](https://user-images.githubusercontent.com/82183018/164754155-cf225d53-5684-4c23-8ce8-1a846900cadd.png)

Está bien así o hay algo que pueda cambiar? Pregunto más que nada para que no se genere confusión a la hora de revisar los TPs aprobados.

Gracias, saludos!
_Franco Parente_

**Comment by gastonprieto** [comment, marked as answer]
Bueans @francoparente 

tenes que ponerle si o si entrega-final, fijate que hay varios issues de gente con un problema similar y esta la explicación de como volver a entregarlo

**Reply by francoparente** [reply]
Listo, gracias!
